### ISSUE TITLE
Hi, I try to use your FFC_BN_ACT

### ISSUE THREAD

**Issue by meiguoofa** [saic-mdal/lama#10]
`

    ffc = FFC_BN_ACT(in_channels=3, out_channels=64, kernel_size=7, padding=0, norm_layer=nn.BatchNorm2d,
                       activation_layer=nn.ReLU, ratio_gin=0, ratio_gout=0, enable_lfu=False,
                       add_out_act='sigmoid')

    x = torch.randn(1, 3, 256, 256)

    reflect = nn.ReflectionPad2d(3)

    x = reflect(x)

    out = ffc(x)

    print("out[0] ", out[1])

`
I try to create a FFC_BN_ACT block, above code is params, are the params set like this correct? it's strange the second output value is a scalar, very much looking forward to your reply !

**Comment by meiguoofa** [comment]
but as you said in your paper image-wide receptive field that covers an entire image in the early layers, The first block in your code [Line367 to Line 269](https://github.com/saic-mdal/lama/blob/a1301cc75ee7a8826b3d9d8c44f900541392d3f7/saicinpainting/training/modules/ffc.py#L367) is just two Conv block? if so,you get the wide receptive field still just rely on two simple convolutional layers ?
